.storybook/components/TokenSpecimen/TokenSpecimen.css

```diff
@@ -23,9 +23,9 @@
   display: inline-block;
   background: var(--eds-theme-color-background-brand-blue-low-emphasis);
   color: var(--eds-theme-color-text-utility-critical);
-  padding: 0.2rem;
+  padding: calc(var(--eds-spacing-size-half) * 1px);
   border-radius: 4px;
-  font-size: 12px;
+  font-size: 0.75rem;
```

Comment:
Where `font-*` is used, we continue to use `rem` units so that browser font-size config continues to work ([docs](https://support.buffer.com/article/512-how-to-adjust-your-browsers-zoom-or-font-size#:~:text=In%20the%20top%20right%20corner,the%20font%20size%20you%20want.))